Box.V2/Managers/BoxMetadataCascadePolicyManager.cs

```diff
@@ -0,0 +1,143 @@
+        /// </summary>
+        /// <param name="policyId">The metadata cascade policy Id to retrieve.</param>
+        /// <param name="fields">Optional fields to retrieve on metadata cascade policy.</param>
+        /// <returns>The metadata cascade policy retrieved by Id.</returns>
+        public async Task<BoxMetadataCascadePolicy> GetCascadePolicyAsync(string policyId, IEnumerable<string> fields = null)
+        {
+            policyId.ThrowIfNullOrWhiteSpace("policyId");
+
+            BoxRequest request = new BoxRequest(_config.MetadataCascadePolicyUri, policyId)
+                .Param(ParamFields, fields);
+
+            IBoxResponse<BoxMetadataCascadePolicy> response = await ToResponseAsync<BoxMetadataCascadePolicy>(request).ConfigureAwait(false);
+
+            return response.ResponseObject;
+        }
+
+        public async Task<BoxCollectionMarkerBased<BoxMetadataCascadePolicy>> GetAllMetadataCascadePoliciesAsync(string folderId, string ownerEnterpriseId = null, int limit = 100, string nextMarker = null, IEnumerable<string> fields = null, bool autopaginate = false)
+        {
+            folderId.ThrowIfNullOrWhiteSpace("folderId");
+
+            BoxRequest request = new BoxRequest(_config.MetadataCascadePolicyUri)
+                .Method(RequestMethod.Get)
+                .Param(ParamFields, fields)
+                .Param("folder_id", folderId)
+                .Param("owner_enterprise_id", ownerEnterpriseId)
+                .Param("limit", limit.ToString())
+                .Param("marker", nextMarker);
+
+            if (autopaginate)
+            {
+                return await AutoPaginateMarker<BoxMetadataCascadePolicy>(request, limit);
+            } else
+            {
+                IBoxResponse<BoxCollectionMarkerBased<BoxMetadataCascadePolicy>> response =
+                    await ToResponseAsync<BoxCollectionMarkerBased<BoxMetadataCascadePolicy>>(request).ConfigureAwait(false);
+
+                return response.ResponseObject;
+            }
+        }
+
+        /// <summary>
+        /// If a policy already exists on the specified folder, this will apply that new policy to the folder and the sub-folder items. 
+        /// </summary>
+        /// <param name="policyId">The policy Id to force apply.</param>
+        /// <param name="conflictResolution">The desired behavior if a conflict exists. Set to either "none" or "overwrite".</param>
+        public async Task<bool> ForceApplyCascadePolicyAsync(string policyId, string conflictResolution)
+        {
+            policyId.ThrowIfNullOrWhiteSpace("policyId");
+            conflictResolution.ThrowIfNullOrWhiteSpace("conflictResolution");
+
+            dynamic jsonObject = new JObject();
+            jsonObject.conflict_resolution = conflictResolution;
+
+            string jsonString = jsonObject.ToString();
+
+            BoxRequest request = new BoxRequest(_config.MetadataCascadePolicyUri, policyId)
```

Comment:
This doesn't look like the correct URL for this endpoint!  Please correct the endpoint here and manually test to ensure that this code works.